tests/components/application-connector/resources/charts/gateway-test/templates/applications/negative-authorisation.yml

```diff
@@ -35,6 +35,44 @@ spec:
             secretName: wrong-oauth-test
             authenticationUrl: "http://{{ .Values.mockServiceName }}.{{ .Values.namespace  }}.svc.cluster.local:8080/v1/api/oauth/token"
             type: OAuth
+    - displayName: mtls-oauth-other-ca
+      name: mtls-oauth-other-ca
+      providerDisplayName: mTLS-OAuth
+      description: Should return 500 for mTLS Oauth with client certificate generated from other CA
+      id: "{{ uuidv4 }}"
+      entries:
+        - type: API
+          targetUrl: "http://{{ .Values.mockServiceName }}.{{ .Values.namespace  }}.svc.cluster.local:8080/v1/api/oauth/ok"
+          centralGatewayUrl: "http://central-application-gateway.kyma-system:8080/negative-authorisation/mtls-oauth-other-ca"
+          credentials:
+            secretName: mtls-oauth-negative-other-ca
+            authenticationUrl: "https://{{ .Values.mockServiceName }}.{{ .Values.namespace  }}.svc.cluster.local:8090/v1/api/mtls-oauth/token"
+            type: OAuthWithCert
+    - displayName: mtls-oauth-incorrect-clientid
```

Comment:
Shouldn't we have also a case for the MTLS one? Like `mtls-incorrect-clientid`

Reply:
There already is `mtls-oauth-incorrect-clientid` for mTLS OAuth and 'wrong-oauth-secret' for OAuth.

Reply:
I mean, you added three negative cases: `mtls-oauth-other-ca`, `mtls-oauth-incorrect-clientid` and `mtls-negative-other-ca`. I asked about adding also `mtls-incorrect-clientid`

Reply:
Well, `clientId` makes sense only for OAuth related stuff. For `mTLS` (aka `CertificateGen`) the secret for Gateway contains certificate and key only.